apis/clusterresources/v1beta1/opensearchegressrules_types.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type OpenSearchEgressRulesSpec struct {
+	ClusterID           string `json:"clusterId"`
+	Name                string `json:"name"`
+	OpenSearchBindingID string `json:"openSearchBindingId"`
+	Source              string `json:"source"`
+	Type                string `json:"type,omitempty"`
+}
+
+type OpenSearchEgressRulesStatus struct {
+	ID     string `json:"id,omitempty"`
+	Status string `json:"status,omitempty"`
```

Comment:
This field is never used. Please delete it

Reply:
fixed